### ISSUE TITLE
Can't boot on 1.21

### ISSUE THREAD

**Issue by Cathekk** [Provismet/VMC-MC#2]
### Type of Error

Crash

### Description

The game crashed whilst initializing game
Error: java.lang.RuntimeException: Could not execute entrypoint stage 'client' due to errors, provided by 'vmc-mc'!

### Mod Version

0.4.1

### Minecraft Version

1.21

### Mod Loader

Fabric

### Steps to Reproduce

![image](https://github.com/user-attachments/assets/f637af79-c82e-4009-82ea-b5503b229f23)

### Crash Log

```bash
The game crashed whilst initializing game
Error: java.lang.RuntimeException: Could not execute entrypoint stage 'client' due to errors, provided by 'vmc-mc'!
```

**Comment by Provismet** [comment]
Can you provide the crash log?

**Comment by Provismet** [comment]
Issue solved as of version 0.4.2
